LICENSE

```diff
@@ -1,8 +1,11 @@
 MIT License
 
-Copyright (c) 2018-2020 audEERING GmbH and Contributors
+Copyright (c) 2018-present audEERING GmbH and Contributors
 
-Author: Hagen Wierstorf
+Author:
+    Christian Geng
```

Comment:
Maybe stick to the same order here as in `[metadata]`?

Reply:
Yeah, maybe not very nice to mix two different order approaches.

Reply:
Should we then order alphabetically or by numbers of contributions?
The first one has the advantage that we never need to change it, but it might give the impression that the first author in the list is the main author.

Reply:
The author contributing most (=you) should be first. Afterwards we could go alphabetically.

Reply:
OK, I have now adjusted the ordering of both files, but I decided to only go after contributions.